components/builder-db/src/config.rs

```diff
@@ -15,5 +15,5 @@
 use num_cpus;
 
 pub fn default_pool_size() -> u32 {
-    (num_cpus::get() * 8) as u32
+    (num_cpus::get() * 2) as u32
```

Comment:
I did! During the course of digging in to PostgreSQL literature, the formula they recommended was `(CPU * 2) + NUM_SPINDLES`. 

The CPU*8 was blowing the default connection limits in postgresql. We could certainly up them (and you know we would in production), but even then I suspect this is the rightish number.